src/python/pants/backend/shell/lint/shfmt/rules.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2021 Pants project contributors.
```

Comment:
No sigificant changes from example-plugin repo, other than using `-l` arg when in fmt mode so that we output the changed files.